base/sort.jl

```diff
@@ -685,16 +685,35 @@ sort(v::AbstractVector; kws...) = sort!(copymutable(v); kws...)
 ## partialsortperm: the permutation to sort the first k elements of an array ##
 
 """
-    partialsortperm(v, k; alg=<algorithm>, by=<transform>, lt=<comparison>, rev=false)
+    partialsortperm(v, k; by=<transform>, lt=<comparison>, rev=false)
```

Comment:
Could also remove the `alg` argument from the `partialsortperm!` docstring.

Reply:
Thank you, done.